Zivid/ReadIterateZDF/ReadIterateZDF.cpp

```diff
@@ -0,0 +1,59 @@
+/*
+This example shows how to import, display and iterate over a Zivid point cloud from a .ZDF
+file.
+*/
+
+#include <Zivid/CloudVisualizer.h>
+#include <Zivid/Zivid.h>
+
+#include <iostream>
+
+int main()
+{
+    try
+    {
+        Zivid::Application zivid;
+
+        std::cout << "Setting up visualization" << std::endl;
+        Zivid::CloudVisualizer vis;
+        zivid.setDefaultComputeDevice(vis.computeDevice());
+
+        std::string Filename = "Zivid3D.zdf";
+        std::cout << "Reading " << Filename << " point cloud" << std::endl;
+        Zivid::Frame frame = Zivid::Frame(Filename);
+
+        std::cout << "Displaying the frame" << std::endl;
+        vis.showMaximized();
+        vis.show(frame);
+        vis.resetToFit();
+
+        std::cout << "Running the visualizer. Blocking until the window closes" << std::endl;
+        vis.run();
+
+        // Extracting point cloud from the frame
+        const auto pointCloud = frame.getPointCloud();
+
+        std::cout << "Point cloud information:" << std::endl;
+        std::cout << "Number of points: " << pointCloud.size() << "\n"
+                  << "Height: " << pointCloud.height() << ", Width: " << pointCloud.width() << std::endl;
+
+        // Iterating over the point cloud and displaying (X, Y, Z, R, G, B, Contrast)
+        for(int i = 0; i < pointCloud.height(); i++)
```

Comment:
I suggest to do `i += 100` instead (and the same for `j`), to avoid extremely long output to console. What do you think?

Reply:
done